packages/web-react/scripts/resolveModuleIds.ts

```diff
@@ -0,0 +1,111 @@
+import * as fs from "fs";
```

Comment:
This script adds `.js` extension to export path to make it accessible via bundlers.